python/tskit/provenance.py

```diff
@@ -72,6 +79,24 @@ def get_environment(extra_libs=None, include_tskit=True):
     return env
 
 
+def get_resources(start_time):
+    # Returns a dict describing the resources used by the current process
+    times = os.times()
+    ret = {
+        "elapsed_time": time.time() - start_time,
+        "user_time": times.user + times.children_user,
+        "sys_time": times.system + times.children_system,
+    }
+    if resource is not None:
+        # Don't report max memory on Windows. We could do this using the psutil lib, via
```

Comment:
This comment isn't accurate for tskit - I don't think we'd want a dependency on psutil.